packages/jest-config/src/readConfigFileAndSetRootDir.ts

```diff
@@ -81,3 +96,35 @@ export default async function readConfigFileAndSetRootDir(
 
   return configObject;
 }
+
+// Load the TypeScript configuration
+const loadTSConfigFile = (configPath: Config.Path): Config.InitialOptions => {
+  // Helper to normalize module's return value
+  const interopRequireDefault = (obj: any): {default: any} =>
+    obj && obj.__esModule ? obj : {default: obj};
+
+  let registerer;
+
+  // Register TypeScript compiler instance
+  try {
+    registerer = require('ts-node').register();
+  } catch (e) {
+    if (e.code === 'MODULE_NOT_FOUND') {
+      throw new Error(
+        `Jest: 'ts-node' is required for the TypeScript configuration files. Make sure it is installed\nError: ${e.message}`,
+      );
+    }
+
+    throw new Error(
+      `Jest: Could not register 'ts-node' to read the TypeScript file: ${configPath}`,
+    );
```

Comment:
```suggestion
    throw error;
```

isn't this easier for consumers to deal with?